src/validateEmail.test.js

```diff
@@ -8,13 +9,38 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof validateEmail('test838@gmail.com')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid email`, () => {
-    expect(validateEmail('test838@gmail.com.'))
-      .toBeTruthy();
+    expect(validateEmail('test838@gmail.com')).toBeTruthy();
+  });
+
+  it(`should return 'false' if email doesn't contain @`, () => {
+    expect(validateEmail('test838.mail.com')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the invalid email`, () => {
+    expect(validateEmail('test838.@gmail.com')).toBeFalsy();
```

Comment:
The test case correctly expects `false` for an email ending with a period. Ensure that this aligns with the task requirements.